src/parser/shared/modules/items/GemChecker.tsx

```diff
@@ -29,216 +32,170 @@ class GemChecker extends Analyzer {
-      if (missingGems > 0) {
-        tooltipContent.push(
-          <Trans id="shared.GemChecker.MissingGemsMultiple">
-            You are missing {missingGems} gems on your {slotName}.
-          </Trans>,
-        );
-      } else {
-        tooltipContent.push(
-          <Trans id="shared.GemChecker.guide.FullyGemmed">
-            {slotName} slots are fully gemmed!
-          </Trans>,
-        );
-      }
-    } else if (ItemHelper.hasBonusId(item, ItemHelper.SINGLE_GEM_BONUS_ID)) {
-      if ((item.gems?.length ?? 0) !== 1) {
-        tooltipContent.push(
-          <Trans id="shared.GemChecker.MissingGemsSingle">
-            You are missing a gem on your {slotName}.
+          <Trans id="shared.GemChecker.MissingSlotsCraftable">
+            <div>
+              You are missing {missingGems} possible gem socket on your {slotName}.
+            </div>
+            <div>
+              Craft/Buy <ItemLink id={ITEMS.MAGNIFICENT_JEWELERS_SETTING.id} /> to add a gem socket.
+            </div>
           </Trans>,
         );
-      } else {
+      } else if (GemChecker.oneAddableGemSlot.includes(slotNumber) && missingGems === 1) {
+        equipmentPerformance = EquipmentPerformance.Potential;
         tooltipContent.push(
-          <Trans id="shared.GemChecker.guide.FullyGemmed">
-            {slotName} slots are fully gemmed!
+          <Trans id="shared.GemChecker.MissingSlotsVault">
+            You do not have a gem socket on your {slotName}. If you don't have good items in your
+            Vault, you can get <ItemLink id={ITEMS.ALGARI_TOKEN_OF_MERIT.id} /> instead and trade 6
+            of them for <ItemLink id={ITEMS.SAD_SOCKET_ADDING_DEVICE.id} /> at the nearby vendor to
+            add a gem socket.
           </Trans>,
         );
       }
     }
 
-    //X Missing Setting
-    if (GemChecker.twoAddableGemSlots.includes(slotNumber)) {
-      const missingGems = 2 - (item.gems?.length ?? 0);
-      if (missingGems > 0) {
-        tooltipContent.push(
-          <Trans id="shared.GemChecker.MissingSlotsCraftable">
-            You are missing {missingGems} possible slot on your {slotName}.<br />
-            Craft/Buy Magnificent Jeweler's Setting to add a slot per.
-          </Trans>,
-        ); //ItemLink in the future when I figure out how they work
-      }
-    } else if (GemChecker.oneAddableGemSlot.includes(slotNumber)) {
-      if ((item.gems?.length ?? 0) !== 1) {
-        tooltipContent.push(
-          <Trans id="shared.GemChecker.MissingSlotsVault">
-            You are missing a possible slot on your {slotName}. <br />6 Algari Token of Merit for
-            S.A.D. to add a slot.
-          </Trans>,
-        ); //ItemLink in the future when I figure out how they work
+    //#region Special Cases
+    if (item.id === ITEMS.CYRCES_CIRCLET.id) {
+      if (missingGems === 0) {
+        equipmentPerformance = EquipmentPerformance.Good;
+      } else {
+        equipmentPerformance = EquipmentPerformance.Good;
```

Comment:
should probably still flag cyrce's with missing gems as a problem. 
```suggestion
        equipmentPerformance = EquipmentPerformance.Fail;
```